ClientApp/viking-vault/src/UserDashboard/TransactionList.tsx

```diff
@@ -30,6 +41,20 @@ class TransactionList extends React.Component<any, IState> {
         transactions: []
     }
 
+    private checkIfUserNull(result:ITransaction[]) {
```

Comment:
A function named checkIfUserNull is telling me that it should return a true/false value. Actually, I see that t returns a list of transactions. Rename the user accordingly and also, specify the return type of the function.